ExtraRoles/ExtraRolesPlugin.cs

```diff
@@ -20,19 +20,14 @@ namespace ExtraRolesMod
     [BepInPlugin(Id)]
     [BepInProcess("Among Us.exe")]
     [BepInDependency(ReactorPlugin.Id)]
```

Comment:
Depend on Essentials, not Reactor (could do both but Essentials already depends on Reactor), plugin load order is dependency -> name sort, if the plugin is renamed and comes before Essentials, it may fail to load.